### PR TITLE
Fixes PC-567 - relays browser user agent in fetch calls

### PR DESCRIPTION
**Pull request by markbrocato**
https://moovweb.atlassian.net/browse/PC-567

### CODE REVIEW

File: packages/react-storefront/src/fetch.js [hunk 1]
```diff
@@ -114,6 +124,8 @@ export default function fetch(url, options = {}, qsOptions) {
     const protocol = url.match(/^https/) ? global.https : global.http
     const { body, ...requestOptions } = createRequestOptions(url, options, qsOptions)
 
+    console.log(requestOptions)
```
Comment (dijs) [inline packages/react-storefront/src/fetch.js]: Did you mean to leave this in?

**Comment by dijs** [comment]
Just needs a change log entry

File: packages/react-storefront/src/fetch.js (hunk 1 above)
Comment (markbrocato) [reply packages/react-storefront/src/fetch.js]: Removed

**Comment by markbrocato** [comment]
Added changelog.

**Pull request merged**